corehq/apps/app_manager/tests/data/suite/child-module-form-workflow-previous.xml

```diff
@@ -26,11 +26,11 @@
     </session>
     <stack>
       <create>
-        <command value="'m0'"/>
```

Comment:
These two commands are just moved to a different part of the stack, which is fine.